mw-webapp/src/component/select/Select.tsx

```diff
@@ -0,0 +1,55 @@
+import {useState} from "react";
+import styles from "src/component/select/Select.module.scss";
+
+export interface OptionType {
+  id: string;
+  value: string;
+  text: string;
+}
+
+/**
+   * Implement a simple select component
+   */
+export interface SelectProps {
+  /**
+   * Label's text
+   */
+  label: string;
+  /**
+   * Default select value (string)
+   */
+  value: string;
+  /**
+   * Select's name (string)
+   */
+  name: string;
+  /**
+   * Options list
+   */
+  options: OptionType[];
+}
+
+export const Select: React.FC<SelectProps> = (props: SelectProps) => {
```

Comment:
how to work with this component? How could I add any handlers to it?

Reply:
Did I understand correctly that I should remove useState() from the component and accept in props selected and onChange()?

Reply:
good idea 

Also use https://www.radix-ui.com/primitives/docs/components/select select based on radix library

Reply:
done